src/gui/transferlistmodel.cpp

```diff
@@ -424,6 +424,8 @@ QVariant TransferListModel::internalValue(const BitTorrent::Torrent *torrent, co
     case TR_SIZE:
         return torrent->wantedSize();
     case TR_PROGRESS:
+        if (alt)
+            return torrent->isPaused() && !torrent->isCompleted();
```

Comment:
`AdditionalUnderlyingDataRole` is still for `column data` but you confuse it by returning no any kind of progress data.